packages/design-tokens/scripts/build.js

```diff
@@ -17,18 +15,20 @@ const DOCS_CONFIG = "./packages/design-tokens/config.designGuidelines.json";
 const configs =
   formatCategory === "docs" ? [DOCS_CONFIG] : [CONFIG, DIST_CONFIG];
 
-configs.forEach(path => {
-  const styleDictionary = StyleDictionary.extend(path);
+import("./formats.mjs").then(({ default: formats }) => {
```

Comment:
Is that the magic 🪄 ?

Reply:
yep, this is it. The trick is that it is a Promise, so I had to find the right place to do it. I tried so many other things.